test/e2e/space_test.go

```diff
@@ -230,7 +230,93 @@ func TestUpdateSpace(t *testing.T) {
 		require.NoError(t, err)
 		tiers.VerifyGivenNsTemplateSet(t, memberAwait, nsTmplSet, tierChecks, tierChecks, tierChecks.GetExpectedTemplateRefs(hostAwait))
 	})
+}
+
+func TestRetargetSpace(t *testing.T) {
+	// given
+	// make sure everything is ready before running the actual tests
+	awaitilities := WaitForDeployments(t)
+	hostAwait := awaitilities.Host()
+	member1Await := awaitilities.Member1()
+	member2Await := awaitilities.Member2()
+	rand.Seed(time.Now().UnixNano())
+
+	t.Run("to no other cluster", func(t *testing.T) {
+		// given
+		space := &toolchainv1alpha1.Space{
+			ObjectMeta: v1.ObjectMeta{
+				Namespace: hostAwait.Namespace,
+				Name:      "oddity-" + strconv.Itoa(rand.Int()),
```

Comment:
btw, why not use `GenerateName`?
```suggestion
				GenerateName:      "oddity-",
```

Reply:
@xcoulon thanks for applying my change - but there are more occurrences like that. It would be maybe worth changing them as well.

Reply:
oh, I was counting on you to submit change proposals for all of them! 😜

Reply:
joke aside, done in b934293